cg/server/admin.py

```diff
@@ -294,6 +294,15 @@ class FlowcellView(BaseView):
     column_filters = ["sequencer_type", "sequencer_name", "status"]
     column_searchable_list = ["name"]
 
+    @staticmethod
+    def view_flow_cell_link(unused1, unused2, model, unused3):
```

Comment:
What are these unused parameters? Can we remove them?

Reply:
In the given code snippet, the variables unused1, unused2, and unused3 are defined as function parameters but are not used within the function body. These variables are included to satisfy the function signature and maintain consistency with other similar functions that might have used those parameters.

However, since they are not used within the function, they can be safely ignored and their values can be left unspecified when calling the view_sample_link method.

Reply:
Since they are not used, maybe we can use an underscore for them? Like `_`, `__` etc. And I don't think `del unused1, unused2, unused3` is necessary